benchmarks/bvh_driver/benchmark_registration.hpp

```diff
@@ -0,0 +1,394 @@
+/****************************************************************************
+ * Copyright (c) 2017-2021 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef BENCHMARK_REGISTRATION_HPP
+#define BENCHMARK_REGISTRATION_HPP
+
+#include <ArborX_CrsGraphWrapper.hpp>
```

Comment:
```suggestion
```

Reply:
I'm not sure. I don't think we guarantee that `ArborX_CrsGraphWrapper.hpp` is included in `ArborX.hpp`.

Reply:
The `query()` free function for a given data structure is provided by its header.  This means all you need is include `<ArborX_LinearBVH.hpp>` or `<ArborX_BruteForce.hpp>`.